src/overlays/actors/ovl_Bg_Breakwall/z_bg_breakwall.c

```diff
@@ -14,80 +23,467 @@ void BgBreakwall_Init(Actor* thisx, GlobalContext* globalCtx);
 void BgBreakwall_Update(Actor* thisx, GlobalContext* globalCtx);
 
 void BgBreakwall_SetupAction(BgBreakwall* this, BgBreakwallActionFunc actionFunc);
+s32 func_808B736C(BgBreakwall* this, GlobalContext* globalCtx);
+s32 func_808B7380(BgBreakwall* this, GlobalContext* globalCtx);
+s32 func_808B73C4(BgBreakwall* this, GlobalContext* globalCtx);
+s32 func_808B73FC(BgBreakwall* this, GlobalContext* globalCtx);
+s32 func_808B7410(BgBreakwall* this, GlobalContext* globalCtx);
+s32 func_808B7460(BgBreakwall* this, GlobalContext* globalCtx);
+s32 func_808B74A8(BgBreakwall* this, GlobalContext* globalCtx);
+s32 func_808B74D8(BgBreakwall* this, GlobalContext* globalCtx);
+s32 func_808B751C(BgBreakwall* this, GlobalContext* globalCtx);
+void func_808B76CC(BgBreakwall* this, GlobalContext* globalCtx);
+void func_808B77D0(BgBreakwall* this, GlobalContext* globalCtx);
+void func_808B77E0(BgBreakwall* this, GlobalContext* globalCtx);
+void func_808B782C(BgBreakwall* this, GlobalContext* globalCtx);
+void func_808B78A4(BgBreakwall* this, GlobalContext* globalCtx);
+void func_808B78DC(BgBreakwall* this, GlobalContext* globalCtx);
+void func_808B7914(BgBreakwall* this, GlobalContext* globalCtx);
+void func_808B7A10(BgBreakwall* this, GlobalContext* globalCtx);
+void func_808B7A90(Actor* thisx, GlobalContext* globalCtx);
+void func_808B7B54(Actor* thisx, GlobalContext* globalCtx);
+void func_808B7D34(Actor* thisx, GlobalContext* globalCtx);
+void func_808B7FE4(Actor* thisx, GlobalContext* globalCtx);
 
-#if 0
 const ActorInit Bg_Breakwall_InitVars = {
-    ACTOR_BG_BREAKWALL,
-    ACTORCAT_ITEMACTION,
-    FLAGS,
-    GAMEPLAY_KEEP,
-    sizeof(BgBreakwall),
-    (ActorFunc)BgBreakwall_Init,
-    (ActorFunc)NULL,
-    (ActorFunc)BgBreakwall_Update,
-    (ActorFunc)NULL,
+    ACTOR_BG_BREAKWALL, ACTORCAT_ITEMACTION,           FLAGS,
+    GAMEPLAY_KEEP,      sizeof(BgBreakwall),           (ActorFunc)BgBreakwall_Init,
+    (ActorFunc)NULL,    (ActorFunc)BgBreakwall_Update, (ActorFunc)NULL,
 };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_808B82E0[] = {
+typedef struct {
+    /* 0x00 */ s16 unk_00;
+    /* 0x04 */ Gfx* unk_04;
+    /* 0x08 */ Gfx* unk_08;
+    /* 0x0C */ AnimatedMaterial* unk_0C;
+    /* 0x10 */ CollisionHeader* unk_10;
+    /* 0x14 */ BgBreakwallUnkFunc unk_14;
+    /* 0x18 */ BgBreakwallUnkFunc2 unk_18;
+    /* 0x1C */ ActorFunc unk_1C;
+} BgBreakwallStruct;
+
+static BgBreakwallStruct D_808B8140[] = {
+    { OBJECT_OMOYA_OBJ, object_omoya_obj_DL_0001A0, NULL, NULL, NULL, func_808B7380, func_808B77D0, NULL },
+    {
+        OBJECT_YUKIMURA_OBJ,
+        object_yukimura_obj_DL_000F98,
+        object_yukimura_obj_DL_000EB0,
+        object_yukimura_obj_Matanimheader_002090,
+        NULL,
+        func_808B73C4,
+        func_808B77D0,
+        NULL,
```

Comment:
It's very inconsistent whether you have a comma on the end or not, which is presumably why it's formatted so badly. Can you remove all of the trailing commas and reformat?

Reply:
This takes up an awful lot of vertical space. What about without, as I originally requested?

Reply:
I don't think it looks very good now, but what do you think?

Reply:
I don't think it looks very good either way, but at least this one doesn't take up nearly so much vertical space. Whoever documents it can worry about getting it perfect, but I think it's good enough as it is now.